weapons.yaml

```diff
@@ -33,4 +33,22 @@
 
     - name: Copy .zshrc template
       command: cp ~/.oh-my-zsh/templates/zshrc.zsh-template ~/.zshrc
+      args:
+        creates: ~/.zshrc
```

Comment:
There is no explicit user here. This will be done for the user executing ansible, which might be confusing. I would put an explicit path here, with the user as a variable : 
```
creates: /home/{{ ansible_ssh_user }}/.zshrc
```

Reply:
It's intended, it's supposed to be done for the user using ansible. That is requested to be an account with passwordless sudo, and some kind of de-facto user account.